src/themes/astro-build.css

```diff
@@ -0,0 +1,166 @@
+	font-display: swap;
+}
+
+html,
+body {
+	overflow: hidden;
+	height: 100vh;
+	margin-top: 0;
+	font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
+		Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+
+	--animation-duration: 500ms;
+	--astro-orange: #ff5d01;
+	--astro-dawn: #f3e9fa;
+	--astro-dusk: #514375;
+	--astro-midnight: #31274a;
+	--astro-midnight-rgb: 49, 39, 74;
+	--astro-blue: #205eff;
+	--astro-pink: #c238bd;
+	--astro-bold-gradient: linear-gradient(
+		to bottom,
+		var(--astro-blue),
+		var(--astro-pink)
+	);
+	--astro-light-gradient: linear-gradient(180deg, #ffb4b4 0%, #c487f0 100%);
+}
+
+[data-twitch-chat] {
+	list-style: none;
+	margin: 0;
+	padding: 0.1em;
+	height: 100vh;
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-end;
+	color: var(--astro-dusk);
+}
+
+[data-twitch-message] {
+	background-color: var(--astro-dawn);
+	margin-bottom: 1em;
+	margin-top: 1em;
+	padding: 0.75em;
+	word-wrap: break-word;
+	box-shadow: 0 19px 53px 0 rgba(var(--astro-midnight-rgb) / 3%),
+		0 10px 28px 0 rgba(var(--astro-midnight-rgb) / 5%),
+		0 6px 16px 0 rgba(var(--astro-midnight-rgb) / 5%),
+		0 3px 9px 0 rgba(var(--astro-midnight-rgb) / 7.5%),
+		0 1px 4px 0 rgba(var(--astro-midnight-rgb) / 1%);
+	transition: opacity var(--animation-duration) ease-in,
+		transform var(--animation-duration) ease-in;
+	opacity: 0;
+	transform: rotateX(-90deg);
+	animation: show var(--animation-duration) 100ms
+		cubic-bezier(0.38, 0.97, 0.56, 0.76) forwards;
+	border-radius: 12px;
+}
+
+@keyframes show {
+	100% {
+		opacity: 1;
+		transform: none;
+	}
+}
+
+.twitch-chat-sender {
+	color: var(--astro-dusk);
+	font-family: 'RT Alias Medium', sans-serif;
+	font-weight: 700;
+	margin-bottom: 0.1em;
+	display: inline;
+}
+
+/*
+[data-twitch-sender-roles*="broadcaster"] .twitch-chat-sender {
+	background-image: var(--astro-bold-gradient);
+	background-clip: text;
+	/* stylelint-disable * /
+	-webkit-background-clip: text;
+	-webkit-text-fill-color: transparent;
+}
+*/
+
+[data-twitch-sender-roles*='broadcaster'] {
+	background-image: var(--astro-light-gradient);
+}
+
+/* [data-twitch-sender-roles*="broadcaster"] * {
+	color: var(--astro-dawn);
+} */
+
+/* [data-twitch-sender-roles*="broadcaster"] [data-twitch-mentioned-user] {
+	background-image: linear-gradient(to right, var(--astro-dawn), var(--astro-dawn));
+} */
+
+[data-twitch-sender-roles*='broadcaster'] .twitch-chat-sender::after {
+	background-image: url('https://astro.build/favicon.ico');
+	background-size: 1.2em 1.2em;
+	height: 1.2em;
+	width: 1.2em;
+	content: '';
+	display: inline-block;
+	margin-bottom: -0.3em;
+}
+
+[data-twitch-message-reply] {
+	display: none;
+}
+
+[data-twitch-mentioned-user] {
+	font-family: 'RT Alias Medium', sans-serif;
+	font-weight: 700;
+	background: transparent;
+	background-image: var(--astro-bold-gradient);
+	background-clip: text;
+	/* stylelint-disable */
```

Comment:
If stylelint gets in the way at any time, feel free to tweak/disable the relevant rules.
The linting is meant to be a helpful guide while developing, a sort of pair-programmer.
One-off disable lines like this are fine ofcourse, but if they are used to silence a message that you would silence in all future encounters, it's time to change the configfile.

The docs are pretty good at explaining what the rule does, and how to change it. In this case, that would be:
https://stylelint.io/user-guide/rules/list/property-no-vendor-prefix/
(if you use VSCode you can go straight to the right page by clicking the name of the error, either in the `problems` tab, or if you have the editor extension, in the popup when you hover over the squiggly)